app/forms/idv/state_id_form.rb

```diff
@@ -43,9 +44,14 @@ def consume_params(params)
         send(:"#{key}=", value)
       end
     end
-
+    
     def raise_invalid_state_id_parameter_error(key)
       raise ArgumentError, "#{key} is an invalid state ID attribute"
     end
+
+    def extra_analytics_attributes(params)
+      { birth_year: params.dig(:dob, :year),
```

Comment:
for the `birth_year` stuff, might be easier to make the base of this PR that branch so we only need to look at the additional changes?